cmd/addimport.go

```diff
@@ -542,7 +552,10 @@ func (p *AddImportParams) createImport() *jwt.Import {
 	if p.service {
 		im.Type = jwt.Service
 		im.Share = p.share
+	} else if p.allowTrace {
+		im.AllowTrace = true
```

Comment:
Are you sure it would not be `im.AllowTrace = p.allowTrace` instead of `true`?

Reply:
Oh sorry... I saw the `else` of the `if p.service` and I thought you were just processing stream properties here and always setting to `true` without seeing that the `else if p.allowTrace`. My bad.